runtime/interpreter/interpreter.go

```diff
@@ -4103,6 +4103,7 @@ func (interpreter *Interpreter) GetCapabilityFinalTargetPath(
 
 func (interpreter *Interpreter) ConvertStaticToSemaType(staticType StaticType) (sema.Type, error) {
 	return ConvertStaticToSemaType(
+		interpreter.memoryGauge,
```

Comment:
```suggestion
		interpreter,
```

Reply:
Passed `interpreter.memoryGauge` to avoid the double dispatch. We should probably do the same in other places